src/components/Board.js

```diff
@@ -16,18 +25,95 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+
+    let boardURL = `${url}${this.props.boardName}/cards`
+
+    axios.get(boardURL)
+      .then((response) => {
+
+        this.props.updateStatusCallback('successfully loaded cards', 'success');
+
+        let cardInfo = response.data.slice(0,100);
+
+
+        let cards = [];
+
+        let i;
+        for (i = 0; i < cardInfo.length; i++) {
+          let card = {};
+
+          card["id"] = cardInfo[i].card.id
+          card["text"] = cardInfo[i].card.text
+          card["emoji"] = cardInfo[i].card.emoji
+
+          cards.push(card);
+        }
+
+        this.setState({cards: cards});
+      })
+      .catch((error) => {
+
+        this.props.updateStatusCallback(error.message, 'error');
+      });
+  }
+
+  deleteCard = (cardID) => {
+
+    let deleteURL = `${url}${this.props.boardName}/cards/${cardID}`;
+    axios.delete(deleteURL);
```

Comment:
It'd probably be best to have `then` and `catch` handlers for this delete request. For example, if the delete fails for some reason, we would want to display that information to the user.